examples/charts/horizon.py

```diff
@@ -0,0 +1,40 @@
+from collections import OrderedDict
+
+import pandas as pd
+import numpy as np
+from bokeh.charts import Horizon
+
+# Here is some code to read in some stock data from the Yahoo Finance API
+AAPL = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=AAPL&a=0&b=1&c=2000",
+    parse_dates=['Date'])
+MSFT = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=MSFT&a=0&b=1&c=2000",
+    parse_dates=['Date'])
+IBM = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=IBM&a=0&b=1&c=2000",
```

Comment:
Use a fixed interval in each case, so that we can reliably test this over time. Preferably use the same interval as other examples that use `ichart`'s API.